user_guide/access_permissions.rst

```diff
@@ -0,0 +1,144 @@
+.. _user-guide-user-management-permissions:
+
+Access and Permissions Management
+=================================
+
+OroCRM provides a very flexible structure of access and permissions management. 
+
+Whether a user will be able to view, create, edit or delete records of a specific entity and change its :term:`Owner` 
+depends on a combination of several factors, namely:
+
+- Ownership Type defined of the entity
+
+- Access Settings defined for the Role assigned to the User
+
+
+.. _user-guide-user-management-permissions-ownership-type:
+
+Ownership Types
+---------------
+
+Each entity in OroCRM has an :ref:`ownership type <user-guide-entity-management-create-other>` type, which defines the 
+level at which permissions will be set for records of the entity.
+
+If the ownership type is set to *"None"*, no authorization is required to see and process the entity and all the users
```

Comment:
Removed the part. It now says: "If the ownership type is set to *"None"*, all the users
within the OroCRM instance will be able to ..."